### DISCUSSION TITLE
[@types/react-aria-modal] - onExit property in missing event type

### DISCUSSION THREAD

**Discussion by eilskn** [DefinitelyTyped/DefinitelyTyped#56655] [Issues with a @types package] [answered]
Hi!

I noticed that the ```event``` type is missing in the ```onExit``` callback, although it is clearly passed there.

https://github.com/davidtheclark/react-aria-modal/blob/master/src/react-aria-modal.js#L113.

I think need to type it because there are times when we want to use ```e.stopPropagation``` or something else.

[a comment by an automated account is not shown]

**Comment by grgr-dkrk** [comment, marked as answer]
Thank you very much. I submitted a PR.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56666
